todoman/ui.py

```diff
@@ -303,6 +305,19 @@ def format_datetime(self, dt):
             self._format_time(time_part)
         )))
 
+    def parse_priority(self, priority):
+        if priority not in ['low', 'high', 'medium', 'none']:
```

Comment:
It should be sufficient to have a single if-else chain here. Also we need to raise a ValueError on invalid values, not return None.